Solutions/Illusive Active Defense/Analytic Rules/Illusive_Detection_Query.yaml

```diff
@@ -0,0 +1,46 @@
+id: 1a7dbcf6-21a2-4255-84b2-c8dbbdca4630
+name: Illusive Incidents Analytic Rule
+description: |
+  'Create a Sentinel incident upon a new Illusive alert (incident) and associate all related Illusive events to the relevant Sentinel alert. This is done by filtering and processing Illusive Syslog messages.'
```

Comment:
Is there any public product documentation that can be added to read more about alerts ? 
I see in sample data `Login event` and `Access event`. will be good refences for analyst to check what those events are.

Reply:
We do not have a public product documentation, we have added the flow in the readme for our customers.
Thanks